Sources/InfomaniakCore/Utils/MatomoUtils.swift

```diff
@@ -0,0 +1,76 @@
+/*
+ Infomaniak Core - iOS
+ Copyright (C) 2023 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import Foundation
+import MatomoTracker
+import SwiftUI
```

Comment:
I think we can have `Matomo` been part of `iOS-core`, but I would advise to keep swift-ui related code in [iOS-core-UI](https://github.com/Infomaniak/ios-core-ui) so we can continue to build this repo without any dependency on any UI frameworks.